spec/features/component_preview_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'rails_helper'
+
+RSpec.describe 'Component preview', :js do
+  it 'loads and initializes stylesheets and javascript' do
+    visit Lookbook::Engine.routes.url_helpers.lookbook_preview_path('one_time_code_input/preview')
```

Comment:
(not to keep adding more work to this PR) but what if we the organized specs by component? then we could have one that checks the clipboard component and clicks it?

Reply:
ex

```ruby
describe 'one_time_code_input' do
  it 'handles one time codes' do
    # ...
  end
end

describe 'clipboard_button' do
  it 'copies stuff' do
    # ...
  end
end